pallets/streaming/src/mock.rs

```diff
@@ -146,12 +146,14 @@ impl pallet_currency_adapter::Config for Test {
 parameter_types! {
     pub const StreamPalletId: PalletId = PalletId(*b"par/strm");
     pub const MaxStreamsCount: u32 = 128;
+    pub const MaxFinisehdStreamsCount: u32 = 2;
```

Comment:
It's the finished streams count or it's canceled or both?

Reply:
StreamLibrary::<T>::try_mutate(account, &StreamKind::Finish, checked_pop)?;

Reply:
It's the finished streams count, include `Cancelled` and `Completed`